setup.py

```diff
@@ -52,18 +53,27 @@ def package_files(directory):
         "entrypoints",
         # Pin sqlparse for: https://github.com/mlflow/mlflow/issues/3433
         "sqlparse>=0.3.1",
+        # Required to run the MLflow server against SQL-backed storage
         "sqlalchemy<=1.3.13",
         "gorilla",
         "prometheus-flask-exporter",
     ],
     extras_require={
         "extras": [
-            "scikit-learn; python_version >= '3.5'",
-            "boto3>=1.7.12",
-            "mleap>=0.16.0",
-            "azure-storage-blob>=12.0",
+            "scikit-learn",
```

Comment:
Can we remove `scikit-learn` here because `dev/extra-ml-requirements.txt` already contains it?

Reply:
I don't think we should remove `scikit-learn`. It's a very common dependency for many of our examples (e.g., on mlflow.org and in the MLflow repo). We also assume that `[extras]` includes scikit-learn in our installation instructions for the MLflow Tutorial: https://mlflow.org/docs/latest/tutorials-and-examples/tutorial.html#what-you-ll-need

Reply:
That makes sense 👍